package.json

```
@@ -86,7 +86,7 @@
         "postcss-scss": "^2.0.0",
         "prettier": "^2.4.1",
         "qunit": "^2.17.2",
-        "raphael": "2.2.0",
+        "raphael": "^2.3.0",
```

It should be exactly 2.3.0
```suggestion
        "raphael": "2.3.0",
```